utilities/rocAL/rocal_external_source/CMakeLists.txt

```diff
@@ -0,0 +1,62 @@
+################################################################################
+#
+# MIT License
+#
+# Copyright (c) 2018 - 2022 Advanced Micro Devices, Inc.
```

Comment:
Please change the copyright from 2022 to 2023

Reply:
@swetha097 Please check this

Reply:
This change has been missed i think
Please change the copyright from 2022 to 2023